### PR TITLE
[O2B-844] Filter according to B field in runs

### PR DESCRIPTION
**Pull request by Jaden505**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [ ] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
-tag colours will now update when looking at run details

Notable changes for developers:
-

Changes made to the database:
-

### CODE REVIEW

File: lib/public/views/Runs/ActiveColumns/runsActiveColumns.js [hunk 1]
```diff
@@ -500,8 +500,8 @@ export const runsActiveColumns = {
     },
     aliceL3AndDipoleCurrent: {
         name: 'L3 / Dipole',
-        visible: false,
+        visible: true,
```
Comment (martinboulais) [inline lib/public/views/Runs/ActiveColumns/runsActiveColumns.js:503]: Do you need this visible true?

File: lib/public/views/Runs/ActiveColumns/runsActiveColumns.js [hunk 2]
```diff
@@ -500,8 +500,8 @@ export const runsActiveColumns = {
...
         filter: ({ aliceL3AndDipoleCurrentFilter }) => selectionDropdown(aliceL3AndDipoleCurrentFilter, { selectorPrefix: 'l3-dipole-current' }),
-        profiles: ['runsPerLhcPeriod', 'runsPerDataPass', 'runsPerSimulationPass'],
+        profiles: ['runsPerLhcPeriod', 'runsPerDataPass', 'runsPerSimulationPass', profiles.none, 'lhcFill', 'environment'],
```
Comment (martinboulais) [inline lib/public/views/Runs/ActiveColumns/runsActiveColumns.js:505]: Why adding 'lhcFill' and 'environment' profiles?

Comment (Jaden505) [reply lib/public/views/Runs/ActiveColumns/runsActiveColumns.js]: Only kept the profiles.none this time

File: lib/public/views/Runs/ActiveColumns/runsActiveColumns.js (hunk 1 above)
Comment (Jaden505) [reply lib/public/views/Runs/ActiveColumns/runsActiveColumns.js]: Set it to false

**Review by martinboulais — CHANGES_REQUESTED** [review]

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: test/public/runs/overview.test.js [hunk 3]
```diff
@@ -596,6 +596,21 @@ module.exports = () => {
         expect(runDurationList.every((runDuration) => runDuration === 'UNKNOWN')).to.be.true;
     });
 
+    it('should successfully apply alice currents filters', async () => {
+        await goToPage(page, 'run-overview');
```
[the remark this answers is not in the document]
Comment (Jaden505) [reply test/public/runs/overview.test.js]: done

File: test/public/runs/overview.test.js [hunk 4]
```diff
@@ -306,7 +306,7 @@ module.exports = () => {
     it('should successfully filter on detectors', async () => {
         // Open filter toggle
         await pressElement(page, '#openFilterToggle');
-        await page.waitForSelector('.detectors-filter .dropdown-trigger');
+        await page.waitForSelector('.detectors-filter .dropdown-trigger', { visible: true });
```
[the remark this answers is not in the document]
Comment (Jaden505) [reply test/public/runs/overview.test.js]: done

**Review by martinboulais — APPROVED** [review]

**Pull request merged**